android-ndk-r11c-api-16-armeabi-c14.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2016, Michele Caini
+# All rights reserved.
+
+if(DEFINED POLLY_ANDROID_NDK_R11C_API_16_ARMEABI_CMAKE_)
+  return()
+else()
+  set(POLLY_ANDROID_NDK_R11C_API_16_ARMEABI_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(ANDROID_NDK_VERSION "r11c")
+set(CMAKE_SYSTEM_VERSION "16")
+set(CMAKE_ANDROID_ARCH_ABI "armeabi")
+set(CMAKE_ANDROID_ARM_MODE TRUE) # 32-bit ARM
+
+polly_init(
+    "Android NDK ${ANDROID_NDK_VERSION} / \
+API ${CMAKE_SYSTEM_VERSION} / ${CMAKE_ANDROID_ARCH_ABI} / \
+32-bit ARM / c++14 support"
+    "Unix Makefiles"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx14.cmake") # before toolchain!
```

Comment:
> android-ndk-r11c-api-16-armeabi-c14.cmake

`c14` is not the same as `cxx14`, see for example https://github.com/ruslo/polly/blob/master/gcc-5-cxx14-c11.cmake

The name of this toolchain should be `android-ndk-r11c-api-16-armeabi-cxx14.cmake`